src/test/java/seedu/address/logic/parser/CountryNoteAddCommandParserTest.java

```diff
@@ -35,23 +42,34 @@ public void parse_withPreamable_throwsParseException() {
     @Test
     public void parse_hasCountryNoNote_throwsParseException() {
         assertThrows(ParseException.class, () -> parser.parse(" c/SG random string"));
+        assertThrows(ParseException.class, () -> parser.parse(" c/SG t/random string"));
+        assertThrows(ParseException.class, () -> parser.parse(" c/SG t/"));
     }
 
     @Test
     public void parse_invalidCountryHasNote_throwsParseException() {
         assertThrows(ParseException.class, () -> parser.parse(" c/Z Z nt/random string"));
+        assertThrows(ParseException.class, () -> parser.parse(" c/Z Z t/a nt/random string"));
+        assertThrows(ParseException.class, () -> parser.parse(" c/Z Z t/ nt/"));
```

Comment:
```suggestion
        assertThrows(ParseException.class, () -> parser.parse(" c/RUS t/a nt/random string"));
        assertThrows(ParseException.class, () -> parser.parse(" c/R t/a nt/random string"));
```
Same reason as below, also, should have a noCountryHasNote test method